src/interfaces/ISablierMerkleBase.sol

```diff
@@ -53,6 +53,9 @@ interface ISablierMerkleBase is IAdminable {
     /// @notice Retrieves the name of the campaign.
     function name() external returns (string memory);
```

Comment:
should we be more explicit with this? since we now have 2 strings, which both are considered "names"

Reply:
`shape` and `ipfsCID` are self explanatory, but IMO `name` can lead to confusion
but, i am ok to keep the current as i don't have a strong opinion

Reply:
I think I understood your point. Also, the DX is better if we use `campaignName` instead of `name`. I am in favour of it.

Reply:
ok, let's use `campaignName`